app/(tabs)/_layout.tsx

```diff
@@ -16,11 +16,17 @@ const TabLayout = () => {
   }
 
   return (
-    <Tabs screenOptions={{ tabBarActiveTintColor: 'blue' }}>
+    <Tabs
+      screenOptions={{
+        headerShown: false,
+        tabBarActiveTintColor: '#f7f7f7',
```

Comment:
Are we going to create a palette or something to avoid hardcoding colours like this?

Reply:
yeah for sure, this is disgusting. I need a way to use the tailwind palette (which I'm not sure is setup properly?) in both classes and components I just haven't looked into it yet.

It's on my list of things to clean up though.